go/ql/lib/semmle/go/frameworks/Beego.qll

```diff
@@ -274,29 +275,11 @@ module Beego {
   }
 
   private class TopLevelTaintPropagators extends TaintTracking::FunctionModel {
-    string name;
-
-    TopLevelTaintPropagators() {
-      this.hasQualifiedName(packagePath(), name) and
-      name in ["HTML2str", "Htmlquote", "Htmlunquote", "MapGet", "ParseForm", "Str2html", "Substr"]
-    }
+    TopLevelTaintPropagators() { this.hasQualifiedName(packagePath(), "MapGet") }
 
     override predicate hasTaintFlow(FunctionInput input, FunctionOutput output) {
-      name in ["HTML2str", "Htmlquote", "Htmlunquote", "MapGet", "Str2html", "Substr"] and
       input.isParameter(0) and
       output.isResult(0)
-      or
-      name = "ParseForm" and
-      input.isParameter(0) and
-      output.isParameter(1)
```

Comment:
Actually I think you should have left this one in, instead of the other one. The signature of `MapGet` is
```
func MapGet(arg1 interface{}, arg2 ...interface{}) (interface{}, [error](https://pkg.go.dev/builtin#error))
```